### ISSUE TITLE
Return Connectome from Connect

### ISSUE THREAD

**Issue by stinebuu** [compneuronmbu/nest-simulator#10]
We should add the possibility of returning a `Connectome` object directly from the `Connect` call:
```
a = nest.Create('iaf_psc_alpha', 10)
c = nest.Connect(a,a, return_connectome=True)
```

This should be quite easy to implement, as we would just need to call `nest.GetConnections(a, a)` if `return_connectome` is `True`.

**Comment by heplesser** [comment]
Had we decided yet on the form of the returned connectome? The current list of lists is just too expensive and inefficient. A NumPy array would be much better, I think. A bit of a challenge here would be that arrays are most efficient if we know in advance how many rows we need.

**Comment by stinebuu** [comment]
This issue is fixed by 85fc32a73fe4088015b3122d266a51b5969150ea. I am thus closing this, as we have #2 to remind us about the internal representation of the connectome.
